src/components/MultiFilterButton/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Link } from 'react-router-dom'
+import { Popover } from 'antd'
+import { ReactComponent as FilterIcon } from '../../assets/filter_icon.svg'
+import { ReactComponent as SelectedIcon } from '../../assets/selected-icon.svg'
+import { ReactComponent as NotSelectedIcon } from '../../assets/not-selected-icon.svg'
+import { ReactComponent as PartialSelectedIcon } from '../../assets/partial-selected-icon.svg'
+import { useSearchParams } from '../../hooks'
+import styles from './styles.module.scss'
+
+export function MultiFilterButton({
+  filteredList,
+  isMobile,
+  filterName,
+}: {
+  filterName: string
+  filteredList: { key: string; value: string; to: string; title: string | JSX.Element }[]
+  isMobile?: boolean
+}) {
+  const params = useSearchParams('tags')
+  const types = params.tags?.split(',').filter(t => t !== '') ?? []
+
+  return (
+    <Popover
+      className={styles.container}
+      placement="bottomRight"
+      trigger={isMobile ? 'click' : 'hover'}
+      overlayClassName={styles.antPopover}
+      content={
+        <div className={styles.filterItems}>
+          <div className={styles.selectTitle}>
+            <h2>Select</h2>
+            {types.length > 0 ? (
+              <>{types.length === filteredList.length ? <SelectedIcon /> : <PartialSelectedIcon />}</>
+            ) : (
+              <NotSelectedIcon />
+            )}
+          </div>
+          {filteredList.map(f => (
+            <Link
+              key={f.key}
+              to={() => {
+                let subTypes = types.map(t => t)
+                if (subTypes.includes(f.value)) {
+                  subTypes = subTypes.filter(t => t !== f.value)
+                } else {
+                  subTypes.push(f.value)
+                }
+                return `${f.to}?${filterName}=${subTypes.slice().join(',')}`
```

Comment:
Queries other than `filterName` are removed in this link.
It should be something as `?${new URLSearchParams({...search, filterName: subTypes.join(',')}).toString()}`